manifests/0000_30_cluster-api_01_credentials-request.yaml

```diff
@@ -266,3 +266,20 @@ spec:
   secretRef:
     name: capv-manager-bootstrap-credentials
     namespace: openshift-cluster-api
+---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-cluster-api-baremetal
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    release.openshift.io/feature-set: "TechPreviewNoUpgrade"
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: Metal3ProviderSpec
+  secretRef:
+    name: capv-manager-bootstrap-credentials
```

Comment:
This needs changing as it conflicts with capv

Reply:
Although it doesn't break payload because it is on different platforms that will never encounter eachother.
We might want to rename (and tombstone?) @cc. @JoelSpeed

Reply:
> Although it doesn't break payload because it is on different platforms that will never encounter eachother.

We have plans in the future to allow multiple platforms to be installed, so, this would then conflict. Names must be unique, so this needs to be fixed